eox_core/api/v1/views.py

```diff
@@ -102,7 +102,7 @@ class EdxappUser(UserQueryMixin, APIView):
     Handles API requests to create users
     """
 
-    authentication_classes = (OAuth2Authentication, SessionAuthentication)
+    authentication_classes = (get_bearer_authentication(), SessionAuthentication)
```

Comment:
Create a constant  and call the function just once in order to avoid to repeat multiple calls. the same comment for the other lines

Reply:
solved